src/connector/src/parser/debezium/mongo_json_parser.rs

```diff
@@ -63,25 +67,43 @@ impl DebeziumMongoJsonParser {
                     )
             })
             .context("Debezium Mongo needs a `_id` column with supported types (Varchar Jsonb int32 int64) in table")?.clone();
-        let _payload_column = rw_columns
-            .iter()
-            .find(|desc| desc.name == "payload" && matches!(desc.data_type, DataType::Jsonb))
-            .context("Debezium Mongo needs a `payload` column with supported types Jsonb in table")?
-            .clone();
 
-        // _rw_{connector}_file/partition & _rw_{connector}_offset are created automatically.
-        if rw_columns
-            .iter()
-            .filter(|desc| desc.is_visible() && desc.additional_column.column_type.is_none())
-            .count()
-            != 2
-        {
-            bail!("Debezium Mongo needs no more data columns except `_id` and `payload` in table");
+        let strong_schema = match &source_ctx.connector_props {
```

Comment:
This does not guarantee that only MongodbCdc properties would be received here, strange.

Reply:
> And there is one more check in the frontend that enforces it to have the only 2 columns [here](https://github.com/risingwavelabs/risingwave/blob/72bbe0aa8a4d3e0c55d39c7e5bc5c08b7472d1ee/src/frontend/src/handler/create_source.rs#L318), which only reads the FORMAT DEBEZIUM_MONGO ENCODE JSON (...); clause.

Then this PR is far from complete then. 🤔

I'll
+ Remove the check
+ Also allow 'strong_schema' placing in the tailing clause

Thanks for your patient and detailed reviews! 😊

Reply:
The CI error is due to `strong_schema` not being part of `KafkaProperties::unknown_fields`. You can refer to #15304 for handling options after `format xxx encode xxx`. Again, it is better to handle different issues in their own PRs.